From 8941017bc0226b60ce306d5271df15820ce66a53 Mon Sep 17 00:00:00 2001
From: Alan Modra <amodra@gmail.com>
Date: Tue, 30 Aug 2016 20:57:32 +0930
Subject: [PATCH] ppc apuinfo for spe parsed incorrectly
Organization: O.S. Systems Software LTDA.

apuinfo saying SPE resulted in mach = bfd_mach_ppc_vle due to a
missing break.

	PR 20531
	* elf32-ppc.c (_bfd_elf_ppc_set_arch): Add missing "break".


Backport from :
https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;a=commitdiff;h=8941017b

Upstream-Status: Backport
Signed-off-by: Fabio Berton <fabio.berton@ossystems.com.br>
---
 bfd/elf32-ppc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/bfd/elf32-ppc.c b/bfd/elf32-ppc.c
index 95ce1dc..e42ef1c 100644
--- a/bfd/elf32-ppc.c
+++ b/bfd/elf32-ppc.c
@@ -2246,6 +2246,7 @@ _bfd_elf_ppc_set_arch (bfd *abfd)
 		case PPC_APUINFO_BRLOCK:
 		  if (mach != bfd_mach_ppc_vle)
 		    mach = bfd_mach_ppc_e500;
+		  break;
 
 		case PPC_APUINFO_VLE:
 		  mach = bfd_mach_ppc_vle;
-- 
2.1.4

